src/components/request.js

```diff
@@ -0,0 +1,19 @@
+export default class Weather {
+  constructor(city, country) {
+    this.apiKey = '1a8dd2ac8b70847191851bae10b18005';
+    this.city = city;
+    this.country = country;
+  }
+
+  async getWeather() {
+    const res = await fetch(`http://api.openweathermap.org/data/2.5/weather?q=${this.city},${this.country}id=524901&appid=${this.apiKey}`);
```

Comment:
## Thumbs up 👍 
✅ Good use of the `async` call function to the `API`. I love that you included the `await` to enforce the fulfilling of the fetch `Promise`. Awesome stuff.